docs/extras/use_cases/question_answering/document-context-aware-QA.ipynb

```diff
@@ -79,7 +79,8 @@
     "chunk_size = 500\n",
     "chunk_overlap = 0\n",
     "text_splitter = RecursiveCharacterTextSplitter(chunk_size=chunk_size, chunk_overlap=chunk_overlap)\n",
-    "all_splits = text_splitter.split_documents(md_header_splits)"
+    "all_splits = text_splitter.split_documents(md_header_splits)\n",
+    "all_metadatas = [{\"source\": \"notion\"}, {\"source\": \"google-docs\"}]"
```

Comment:
Nit: we can actually remove `all_metadatas` now b/c the splitter populates it for us.